src/components/Input/index.ts

```diff
@@ -0,0 +1,119 @@
+import Block from '../../utils/Block';
+import template from './input.pug';
+import * as inputStyles from './input.module.scss';
+import { Icon } from '../Icon';
+import { ValidationResult } from '../../utils/CustomValidation';
+import refElementsCollection from '../../utils/RefElementsCollection';
+
+interface InputProps {
+  title: string,
+  type: string,
```

Comment:
тут можно типизировать детальнее, например так:
```suggestion
  type: 'text' | 'number' | 'email' и т.д.
```